book/version_control.md

```diff
@@ -435,13 +431,14 @@ Once a PR has been reviewed and the reviewer is happy with the changes, the Conv
 The repository settings can be adjust to project branches against specific actions. To enforce peer review, you should consider preventing merging onto the `main` branch without an approved Pull Request. Combining this with a [Pull Request template](https://docs.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository) ensures that a standard peer review process is followed for all changes. 
 ```
 
-
 (forking)=
-### Forking
+
+### Fork a repository
```

Comment:
Needs a little more detail on when or why, otherwise the action doesn't make sense

Reply:
Updated to:
### Fork a repository to maintain an independent copy